src/modules/typescript.rs

```diff
@@ -0,0 +1,63 @@
+use ansi_term::Color;
+use std::process::Command;
+
+use super::{Context, Module};
+
+/// Creates a module with the current typescript version
+///
+/// Will display the typescript version if any of the following criteria are met:
+///     - Current directory contains a `.ts` file
+///     - Current directory contains a `tsconfig.json` file
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let is_ts_project = context
+        .new_scan_dir()
+        .set_files(&["tsconfig.json"])
+        .set_extensions(&["ts"])
+        .scan();
+
+    if !is_ts_project {
+        return None;
+    }
+
+    match get_typescript_version() {
+        Some(typescript_version) => {
+            const TYPESCRIPT_CHAR: &str = "■ ";
```

Comment:
You can probably use the 🇹 emoji, can't you? :thinking:

Reply:
I've tried it and this is how it looks.
<img width="412" alt="Screen Shot 2019-08-20 at 20 50 22" src="https://user-images.githubusercontent.com/42851547/63371140-30054400-c38c-11e9-91d8-5c1ccb6df9bf.png">
If that's better in your opinion I can commit the changes 😄

Reply:
aah, I guess font support might be an issue. I'm not sure which to choose in that case, maybe @matchai can decide?

Reply:
@youssefhabri I like the square better